src/test/java/com/wikia/webdriver/testcases/discussions/FollowingPostTests.java

```diff
@@ -138,38 +135,33 @@ public void userOnMobileCanUnfollowPostOnUserPostsPage() {
   /**
    * By default all posts on "Followed" tab are followed.
    */
-  @Test(groups = "discussions-userMobileFollowingPost", enabled = false)
+  @Test(groups = MOBILE, enabled = false)
   @Execute(asUser = User.USER)
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
   @RelatedIssue(issueID = "SOC-3674", comment = "Introducing pagination")
   public void userOnMobileCanFollowAndUnfollowPostOnFollowedPostsPage() {
     createPostAsUserRemotely();
     final FollowPage page = FollowPage.open();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "page" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)